### PR TITLE
Replace structopt by clap (latency crate)

### PR DESCRIPTION
**Pull request by eddie9712**

### CODE REVIEW

**Review by william-wyliang — CHANGES_REQUESTED** [review]
The commits 563baf7 and 505fd7f are duplicated commits in previous PR.

**Review by YuanYuYuan — APPROVED** [review]

File: latency/Cargo.toml [hunk 1]
```diff
@@ -38,7 +38,7 @@ env_logger = "0.9.0"
 log = "0.4.14"
 rand = "0.8.3"
 slab = "0.4.2"
-structopt= "0.3.21"
+clap = { version = "3.1.1", features = ["derive"] }
```
Comment (william-wyliang) [inline latency/Cargo.toml]: Consider to use version 3.1.5

Comment (eddie9712) [reply latency/Cargo.toml]: Upgraded.

File: latency/Cargo.toml [hunk 2]
```diff
@@ -38,7 +38,7 @@ env_logger = "0.9.0"
 log = "0.4.14"
 rand = "0.8.3"
 slab = "0.4.2"
-clap = { version = "3.1.1", features = ["derive"] }
+clap = { version = "3.1.5", features = ["derive"] }
```
Comment (william-wyliang) [inline latency/Cargo.toml:41]: This small change is actually a tiny uncompleted part of the previous commit. Typically it won't be an independent commit. Suggest to amend the change in the previous commit to make the log cleaner.

**Review by william-wyliang — CHANGES_REQUESTED** [review]

File: latency/Cargo.toml (hunk 2 above)
Comment (eddie9712) [reply latency/Cargo.toml:41]: Ok, commits squashed.

**Review by william-wyliang — APPROVED** [review]
